README.md

```diff
@@ -17,21 +17,24 @@ the square brackets (for multiple-choice questions).
 
 Select all that apply:
 ```
-[] Strings
-[] Booleans
-[] Undefined
+[X] Strings
+[X] Booleans
+[X] Undefined
 [] NaN
-[] Integers
+[X] Integers
 [] Arrays
```

Comment:
:+1:  You are correct on all the basic data types, I would classify `Arrays` as a complex data type